opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -146,20 +176,12 @@ message SummaryTimeSeries {
   // The set of label values that uniquely identify this timeseries. Applies to
   // all points. The order of label values must match that of label keys in the
   // metric descriptor.
-  repeated LabelValue label_values = 1;
+  LabelValues label_values = 1;
 
   // The data points of this timeseries.
   repeated SummaryValue points = 2;
 }
 
-message LabelValue {
-  // The value for the label.
-  string value = 1;
-  // If false the value field is ignored and considered not set.
-  // This is used to differentiate a missing label from an empty string.
-  bool has_value = 2;
```

Comment:
do we have to differentiate missing value from empty string at a first place? How important is it?

Reply:
This is explicitly required in the API: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-metrics-user.md#missing-label-keys

>When the SDK interprets a LabelSet in the context of grouping aggregated values for an exporter, and where there are keys that are missing, the SDK is required to consider these values explicitly unspecified, a distinct value type of the exported data model.

Another way this could be modeled in `Metric` data structure is to require every metric timeseries that has a different set of labels with defined values to be stored in a separate instance of `Metric` which would point to `MetricDescriptor` containing label_keys only for defined labels. Hard to tell which approach would be more efficient, it all depends on statistical composition of data.

For now in this change I tried not to change existing functionality but simply modeled it in a slightly more efficient way.

Reply:
OpenMetrics does not make this separation. Not 100% sure if it is needed or not to separate between empty string and not present.